gamemodes/terrortown/gamemode/server/sv_corpse.lua

```diff
@@ -16,6 +16,25 @@ local ConVarExists = ConVarExists
 local CreateConVar = CreateConVar
 local hook = hook
 
+-- If detective mode, announce when someone's body is found
+local cvBodyfound = CreateConVar("ttt_announce_body_found", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
+local cvRagCollide = CreateConVar("ttt_ragdoll_collide", "0", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
+local cvDeteOnlyConfirm = CreateConVar("ttt2_confirm_detective_only", "0", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
+local cvDeteOnlyInspect = CreateConVar("ttt2_inspect_detective_only", "0", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
+
+hook.Add("TTT2SyncGlobals", "TTT2SyncCorpseGlobals", function()
+	SetGlobalBool(cvDeteOnlyConfirm:GetName(), cvDeteOnlyConfirm:GetBool())
+	SetGlobalBool(cvDeteOnlyInspect:GetName(), cvDeteOnlyInspect:GetBool())
+end)
+
+cvars.AddChangeCallback(cvDeteOnlyConfirm:GetName(), function(name, old, new)
+	SetGlobalBool(cvDeteOnlyConfirm:GetName(), tobool(new))
+end, cvDeteOnlyConfirm:GetName())
+
+cvars.AddChangeCallback(cvDeteOnlyInspect:GetName(), function(name, old, new)
+	SetGlobalBool(cvDeteOnlyInspect:GetName(), tobool(new))
+end, cvDeteOnlyInspect:GetName())
```

Comment:
Global vars are not needed anymore, as the CVAR syncing bug was fixed. So i'd say remove the Global vars and just use the convar instead.